lib/mix/lib/mix/tasks/release.init.ex

```diff
@@ -245,11 +245,11 @@ defmodule Mix.Tasks.Release.Init do
     popd
 
     if not defined RELEASE_NAME (set RELEASE_NAME=<%= @release.name %>)
-    if not defined RELEASE_VSN (for /f "tokens=1,2" %%K in (!RELEASE_ROOT!\releases\start_erl.data) do (set ERTS_VSN=%%K) && (set RELEASE_VSN=%%L))
+    if not defined RELEASE_VSN (for /f "tokens=1,2" %%K in ('type "!RELEASE_ROOT!\releases\start_erl.data"') do (set ERTS_VSN=%%K) && (set RELEASE_VSN=%%L))
```

Comment:
The [`FOR` command](https://ss64.com/nt/for.html) also accepts a string when the `/f` option is passed, so just quoting the path was not enough.

I got the `type` _trick_ from here: http://www.celticproductions.net/articles/12/batch-files/file-path-and-name-with-spaces-in-batch-file-for-loop.html